test/client.test.js

```diff
@@ -412,6 +412,56 @@ describe('Client', () => {
       requiredKeys.every(key => driverEnvHeader.includes(key))
     ).toBeDefined()
   })
+
+  test('http2SessionIdleTime env overrides client config', async () => {
```

Comment:
- add a test for passing garbage into the env var - something that is not `Infinity` and not an integer.
- add a test for pssing garbage in the options
- add a test for passing a negative number in the env var
- add a test for passing a negative number in the options

Reply:
These should all be covered now.